manim/_config/utils.py

```diff
@@ -85,7 +87,7 @@ def make_config_parser(custom_file: str = None) -> configparser.ConfigParser:
 
     Parameters
     ----------
-    custom_file : :class:`str`
+    custom_file : Union[:class:`str`, :class:`os.PathLike`], optional
```

Comment:
If there are actual type hints in the code, the type hints added in the docstring should be removed. (There are many places in the library where this has not been done yet.)